core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java

```diff
@@ -273,6 +274,10 @@ void append() throws CommitFailedException {
                         row.putTimestamp(colIndex, buffer.readLong(offset));
                         offset += Long.BYTES;
                         break;
+                    case LineTcpParser.ENTITY_TYPE_UUID:
+                        row.putUuid(colIndex, buffer.readLong(offset), buffer.readLong(offset + Long.BYTES));
```

Comment:
i have seen this somewhere, if we have `row.putUuid(CharSequence)` we could parse in one place

Reply:
ILP handlers parses UUIDs here: https://github.com/jerrinot/questdb/blob/2042d4f25748537f3a94fd4f104ec667f7df1fd7/core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpEventBuffer.java#L240-L251

we do it early for 2 reasons:
1. storing UUIDs in the event objects as long saves space in the buffer
2. we can log a helpful error: `cast error for line protocol` - postponing to the Row copiers would muddy it